### DISCUSSION TITLE
Interest in adding new endf6 libraries ?

### DISCUSSION THREAD

**Discussion by nplinden** [luca-fiorito-11/sandy#319] [General]
Hi all,
I've been using sandy to process file from the TENDL-2023 and JEFF-3.1.1 libraries. Is there any interest in adding these to the `libraries.py` file to allow users to easily download them?

I particular I noticed IAEA is used as a source for all libraries, but they do not offer TENDL-2023 yet, would it be fine to add the libraries using [PSI's website](https://tendl.web.psi.ch/tendl_2023/tendl2023.html) as a source ?

I have these changes in a personal branch and can create a PR is needed, just wanted to make sure there is interest for this beforehand.

**Comment by luca-fiorito-11** [comment]
This would be appreciated!
As long as the IAEA website is used as a source...the pre-implemented functions/method in sandy should work.
it is just a matter of adding the right file paths and names.

For TENDL, probably something else should be done
Then, I would expect separate pull requests.

**Reply by nplinden** [reply]
Great, I created three PRs to implement this: one that fix an issue with delimiters that prevented some JEFF-3.1.1 nuclides from being processed correctly, and two with the correct URLs to fetch the files from IAEA and PSI.

**Comment by luca-fiorito-11** [comment]
PR's #321 and #322 were merged.
These give access to JEFF-3.1.1 and TENDL-2023 xs data.

**Reply by nplinden** [reply]
Thanks for the merge, it should be noted that some files in JEFF-3.1.1 won't be processed by sandy because they contain "?" characters, this is fixed with PR #318
